app/lib/actions/katello/host/upload_package_profile.rb

```diff
@@ -51,6 +47,13 @@ def run
             end
           end
         end
+
+        def run
+          profile = JSON.parse(input[:profile_string])
```

Comment:
Dont think we want to delete what is in the input. Because that is used by foreman-tasks for logging.

Reply:
@parthaa notice it clears the input down below.  So i think @johnpmitsch's suggestion make sense, although we probably want to leave a comment and the 'TRIMMED' piece as well so that we know this was executed.  So i think .delete() makes sense semantically , but i think we should leave it as.